src/main/scala/fr/brouillard/gitbucket/h2/controller/H2BackupController.scala

```diff
@@ -42,26 +38,32 @@ class H2BackupController extends ControllerBase with AdminAuthenticator {
   })
 
   get("/api/v3/plugins/database/backup") {
+    doBackupMoved()
+  }
+
+  post("/api/v3/plugins/database/backup") {
     context.loginAccount match {
       case Some(x) if (x.isAdmin) => doExport()
       case _ => org.scalatra.Unauthorized()
     }
   }
 
+  // Legacy api that was insecure/open by default
   get("/database/backup") {
-    if (sys.props.get("secure.backup") exists (_ equalsIgnoreCase "true"))
-      org.scalatra.TemporaryRedirect("/api/v3/plugins/database/backup?dest=" + params.getOrElse("dest", defaultBackupFileName()))
-    else {
-      doExport()
-    }
+    doBackupMoved()
+  }
+
+  private def doBackupMoved(): ActionResult = {
+    org.scalatra.MethodNotAllowed("This has moved to POST /api/v3/plugins/database/backup")
   }
 
-  private def doExport(): Unit = {
+  private def doExport(): ActionResult = {
```

Comment:
This method is now used by only `post("/api/v3/plugins/database/backup")`. No longer necessary as an independent method.